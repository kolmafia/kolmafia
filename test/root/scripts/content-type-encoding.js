module.exports.main = function() { var k = require("kolmafia"); k.print(k.Item.get("Our Daily Candlesâ„¢ order form")) }
