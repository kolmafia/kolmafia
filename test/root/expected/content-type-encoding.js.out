Our Daily Candles&trade; order form
